README.md

```diff
@@ -45,6 +45,8 @@ $ cd parabol
 $ cp .env.example .env # Add your own vars here
 $ yarn
 $ yarn db:start
+$ yarn pg:build
+$ yarn relay:build
```

Comment:
-1 this is part of `yarn dev`
https://github.com/ParabolInc/parabol/blob/ae60b038dc875eafa78eaf64058361c7ab846f96/pm2.dev.config.js#L71
https://github.com/ParabolInc/parabol/blob/ae60b038dc875eafa78eaf64058361c7ab846f96/pm2.dev.config.js#L52
If it does not work, it needs fixing there.

Reply:
What error are you getting? If it's just a race condition and it would work after waiting some time, I'm fine with leaving it in here.

Reply:
It isn't working for me at least 😿

Reply:
https://www.loom.com/share/d563065bc1ef43f988d8f515c5fb7713

Reply:
What error message are you getting?

Reply:
This change is discarded but please check the video I shared https://www.loom.com/share/d563065bc1ef43f988d8f515c5fb7713

Reply:
Sorry, missed the comment before. I'll take a look